src/vs/platform/update/electron-main/abstractUpdateService.ts

```diff
@@ -93,7 +86,7 @@ export abstract class AbstractUpdateService implements IUpdateService {
 	*/
 	protected async initialize(): Promise<void> {
 		// --- Start Positron ---
-		const updateChannel = process.env.POSITRON_UPDATE_CHANNEL ?? UpdateChannel.Prereleases;
+		const updateChannel = process.env.POSITRON_UPDATE_CHANNEL ?? this.configurationService.getValue<string>('update.positron.channel');
```

Comment:
Would recommend emitting a log statement when we ignore the user-set value in favor of the environment variable.

Reply:
Good idea!